src/Data/Constraint/Deriving/ToInstance.hs

```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE CPP                #-}
```

Comment:
Maybe we can add `mkVisFunTys` definition to `CorePluginM.hs` to avoid CPP in this file?